boards/rcar_salvator_xs_m3.overlay

```diff
@@ -19,11 +19,19 @@
 };
 
 / {
+	firmware {
+		tee {
+		compatible = "linaro,optee-tz";
+		method = "smc";
+		status = "okay";
+		};
+	};
+
 	fstab {
 		compatible = "zephyr,fstab";
 		storage: storage {
 			compatible = "zephyr,fstab,littlefs";
-			mount-point = "/tmp";
+			mount-point = "/tee";
```

Comment:
This is simply wrong. /tmp mountpoint was created for purpose and it is used in the system. I think you should discuss this with @al1img

Reply:
This change removed